jaseci_ai_kit/jaseci_ai_kit/modules/ph/inference.py

```diff
@@ -45,19 +63,26 @@ def __init__(self, config: Dict, uuid: str = None):
         self.model.eval()
 
         # Initialize Pre-processor
+        preprocessor_args = self.infer_config["preprocess"].get("args", {})
+        if self.infer_config["preprocess"]["type"].startswith("Custom"):
+            preprocessor_args["module_name"] = "CustomPreProcessor"
         self.preprocessor = getattr(
             process_module, self.infer_config["preprocess"]["type"]
-        )(**self.infer_config["preprocess"].get("args", {}))
+        )(**preprocessor_args)
         # Initialize Post-processor
+        postprocessor_args = self.infer_config["postprocess"].get("args", {})
+        if self.infer_config["postprocess"]["type"].startswith("Custom"):
+            postprocessor_args["module_name"] = "CustomPostProcessor"
         self.postprocessor = getattr(
             process_module, self.infer_config["postprocess"]["type"]
-        )(**self.infer_config["postprocess"].get("args", {}))
+        )(**postprocessor_args)
 
     @torch.no_grad()
     def predict(self, data: Any) -> Any:
         data = self.preprocessor.process(data)
         data = data.to(self.device)
         output = self.model(data)
+        self.logger.info("Predict is called")
```

Comment:
Remove debugging prints.

Reply:
These are not debugging logs its for logging for each ph